opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/ScopeManagerShim.java

```diff
@@ -30,7 +30,12 @@ public ScopeManagerShim(Tracer tracer) {
 
   @Override
   public Span activeSpan() {
```

Comment:
So activeSpan returns null if no real Span is active? Is this the expected behavior? Please comment accordingly if that is the case.

Reply:
That's very correct. We never had a no-op `Span` here (although it was mentioned at some point ;) )

Reply:
Can you put a comment in the code that reflects this? Then we can merge this PR.